Makefile

```diff
@@ -33,3 +39,37 @@ init:
 
 update:
 	@echo "'make update' has been removed. Use 'make install' instead."
+
+test:
+	make lint && make unittest && make uitest
+
+uitest:
+	make uitest-phantom && make uitest-slimer
+
+uitest-phantom: css templates
```

Comment:
On a local branch, I removed the ```css templates``` dependency directives. Starting up the webserver will do this so we shouldn't need to do it all the time. It takes about 2-3 seconds to do each time so it slows down testing.